### PR TITLE
DALA-459 dependency updates

### PR DESCRIPTION
**Pull request by MarcTM01**
# Manual Maintenance Sprint [NR]
# Maintenance tasks (to be completed by the assignee)
## Skyminder
- [x] Complete manual maintenance in the Skyminder Repo
- [x] Release a new version `0.1.8` (Please put new version number here)
## EDC
- [x] Complete manual maintenance in the EDC Repo
- [ ] ~Release a new version `x.x.x` (Please put new version number here)~
### Gradle update
- [x] Execute `gradlew dependencyUpdates` to get a report on Dependencies with updates
- [x] Update `settings.gradle.kts` (for libraries), `build.gradle.kts` (for plugins) and `gradle.properties` (for jacoco and ktlint)
Note: fasterXML is managed by spring and ktlint by jlleitschuh, thus NO manual version update should be conducted
- [x] update the gradle wrapper: execute `gradle wrapper --gradle-version X.Y.Z`

### Dataland frontend
- [x] Update node version in `dataland-frontend/build.gradle.kts`
- [x] Update node packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json
- [x]   Run the `updatepackagelock`, e.g. by  `npm run updatepackagelock` script to update `package-lock.json` and check for security issues
  (Known issues appeared in the past with updating Jest, openApiGenerator and Eslint).

### Dataland keycloak theme
- [x] Update node version in `dataland-keycloak/dataland_theme/login/build.gradle.kts`
- [x] Update node packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json
- [x]   Run the `updatepackagelock`, e.g. by  `npm run updatepackagelock` script to update `package-lock.json` and check for security issues
  (Known issues appeared in the past with updating Jest, openApiGenerator and Eslint).

### Dockerfile updates
Update versions in the following dockerfiles
- [x] `./baseDockerfiles/cypressBaseImageDockerfile`
  - [x] On any change run the corresponding job in GitHub
- [x] `./baseDockerfiles/temurinBaseImageDockerfile`
  - [ ] ~On any change run the corresponding job in GitHub~
- [x] `./dataland-backend/Dockerfile`
- [x] `./dataland-keycloak/Dockerfile`
- [x] Update versions in the `docker-compose.yml` file
## Server updates
Execute `sudo apt-get update && sudo apt-get upgrade` on
- [x] dev-dataland.duckdns.org
- [x] dataland-tunnel.duckdns.org
- [x] dataland-letsencrypt.duckdns.org
- [ ] ~(OPT) dataland.com~

## Conclusion
- [x] After updating all components check if everything still works
  - [x] Document any conflicts and skipped update by placing comments on this PR
- [x] The new version is deployed to the dev server using this branch and real data
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] It's verified that real data has been used
  - [x] It's verified that everything seems to be working fine by manually using the website
  - [x] All implemented Social Logins have been tested manually in the UI

# Review (to be completed by the reviewer)
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github.
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described above
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] It's verified that everything seems to be working fine by manually using the website
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

**Comment by MarcTM01** [comment]
Skipped update of detekt to 1.21.0 due to incompatibility with kotlin 1.7.10

**Comment by MarcTM01** [comment]
Skipped update of ts-jest to 28.0.8 because @vue/cli-plugin-unit-jest still needs ^27.0.4 (Ref https://github.com/vuejs/vue-cli/issues/7129)

File: .github/workflows/CD.yaml [hunk 1]
```diff
@@ -103,7 +103,6 @@ jobs:
           TRUSTEE_BASE_URL: ${{ secrets.TRUSTEE_BASE_URL }}
           TRUSTEE_ENVIRONMENT_NAME: ${{ github.event.inputs.trusteeEnvironmentName }}
           SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
-          TRUSTEE_CREDENTIALS: ${{ secrets.TRUSTEE_CREDENTIALS }}
```
Comment (MarcTM01) [inline .github/workflows/CD.yaml:106]: Im Nachinen: Das ist eigentlich Teil von DALA-297 und sollte (und wird auch) in dem Branch bereits gelöscht

**Comment by MarcTM01** [comment]
Reverted Primevue update to 1.16 because of issue with AutoComplete component (An issue with PrimeVue has been filed under https://github.com/primefaces/primevue/issues/2881)

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Manual Maintenance Sprint [NR]
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:1]: Mit dem File wird der Abschnitt in der Readme obsolet - wir sollten das nicht 2x pflegen.

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- [ ] Complete manual maintenance in the Skyminder Repo
+- [ ] Release a new version `x.x.x` (Please put new version number here)
+## EDC
+- [ ] Complete manual maintenance in the EDC Repo
+- [ ] Release a new version `x.x.x` (Please put new version number here)
+## Dataland
+### Gradle update
+- [ ] Execute `gradlew dependencyUpdates` to get a report on Dependencies with updates
+- [ ] Update `settings.gradle.kts` (for libraries), `build.gradle.kts` (for plugins) and `gradle.properties` (for jacoco and ktlint)
+Note: fasterXML is managed by spring and ktlint by jlleitschuh, thus NO manual version update should be conducted
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:13]: der Teil mit jlleitschuh raus

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- [ ] update the gradle wrapper: execute `gradle wrapper --gradle-version X.Y.Z`
+
+### Dataland frontend
+- [ ] Update node version in `dataland-frontend/build.gradle.kts`
+- [ ] Update node packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json
+- [ ]   Run the `updatepackagelock`, e.g. by  `npm run updatepackagelock` script to update `package-lock.json` and check for security issues
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:19]: wenn ich `npm run updatepackagelock` ausführe verändert sich meine package-lock

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  (Known issues appeared in the past with updating Jest, openApiGenerator and Eslint).
+
+### Dataland keycloak theme
+- [ ] Update node version in `dataland-keycloak/dataland_theme/login/build.gradle.kts`
+- [ ] Update node packages: run the `updatepackages` script, e.g. by  `npm run updatepackages` to update versions in package.json
+- [ ]   Run the `updatepackagelock`, e.g. by  `npm run updatepackagelock` script to update `package-lock.json` and check for security issues
```
Comment (ppuetsch) [inline .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:25]: wenn ich npm run updatepackagelock ausführe verändert sich meine package-lock

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 4 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:19]: Vielleicht liegt das daran, dass node/npm in der aktuellen version andere keys in die package-lock schreiben. Ich nutze node 18.7.0 - gebundled mit npm 8.15.0

Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:19]: image

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 2 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:1]: ok

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 3 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md]: ok

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 4 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:20]: ok

File: .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md (hunk 5 above)
Comment (ppuetsch) [reply .github/PULL_REQUEST_TEMPLATE/manual_maintenance.md:26]: ok

**Pull request merged**